docs/en/manuals/live-update.md

```diff
@@ -23,102 +31,32 @@ To have the engine load such a collection dynamically, we can simply add a colle
 
 This is done by simply checking the *Exclude* checkbox in the collection proxy properties, telling the bundler to leave any content in *monalisa.collection* out when creating an application bundle.
 
+::: important
+Any resources referenced by the base game package, will not be excluded.
+:::
+
```

Comment:
I converted notes to the standard "sidenote" ot "important" markup we use elsewhere in the manuals